test/functional/rpc_signrawtransactionwithkey.py

```diff
@@ -46,16 +49,12 @@
 
 class SignRawTransactionWithKeyTest(BitcoinTestFramework):
     def set_test_params(self):
-        self.setup_clean_chain = True
         self.num_nodes = 2
 
     def send_to_address(self, addr, amount):
-        input = {"txid": self.nodes[0].getblock(self.block_hash[self.blk_idx])["tx"][0], "vout": 0}
-        output = {addr: amount}
-        self.blk_idx += 1
-        rawtx = self.nodes[0].createrawtransaction([input], output)
-        txid = self.nodes[0].sendrawtransaction(self.nodes[0].signrawtransactionwithkey(rawtx, [self.nodes[0].get_deterministic_priv_key().key])["hex"], 0)
-        return txid
+        script_pub_key = address_to_scriptpubkey(addr)
+        tx = self.wallet.send_to(from_node=self.nodes[0],scriptPubKey=script_pub_key, amount=int(amount * COIN))
+        return [tx["txid"], tx["sent_vout"]]
```

Comment:
It might be better to return a tuple here
```suggestion
        return tx["txid"], tx["sent_vout"]
```